### PR TITLE
Raindrop.io update - adds Detail view

### PR DESCRIPTION
**Pull request by lardissone**
## Description

Adds Detail view to Raindrop.io extension.

## Screencast

Here's an example on this ticket: https://github.com/raycast/extensions/issues/2203#issuecomment-1179096742

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by lardissone** [comment]
Hi @danielsequeira !

Ah yes, didn't noticed about this change!
Just added screenshots and also changelog!

BTW, it would be great to implement something (probably same @raycastbot) that let creator know their extensions are outdated (missing metadata, or similar).

File: extensions/raindrop/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Raindrop.io Extension Changelog
+
+## [Add details view] - 2022-07-13
+
+- Adds option to commands to open a detail view of the bookmark
```
Comment (danielsequeira) [inline extensions/raindrop/CHANGELOG.md:5]:
```suggestion
- Adds action to open a detail view of the bookmark
```

File: extensions/raindrop/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Raindrop.io Extension Changelog
+
+## [Add details view] - 2022-07-13
```
Comment (danielsequeira) [inline extensions/raindrop/CHANGELOG.md:3]:
```suggestion
## [Add details view] - 2022-07-14
```

**Comment by danielsequeira** [comment]
Hi @lardissone,

Thank you so much for the changes, I just did another check and suggested two more tiny changes. After that we should be good to merge it.

Also thank you for the suggestion, definitely something that could help a lot. I will share it internally.

**Pull request merged**